app/assets/javascripts/app/components/notification_card/notification_card.js

```diff
@@ -0,0 +1,37 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.components')
+    .directive('notificationCard', notificationCard)
+    .controller('NotificationCardCtrl', NotificationCardCtrl);
+
+  function notificationCard() {
+    return {
+      restrict: 'E',
+      scope: {
+        notification: '='
+      },
+      replace: true,
+      templateUrl: 'components/notification_card/templates/notification_card.html',
+      controller: 'NotificationCardCtrl',
+      controllerAs: 'ctrl',
+      bindToController: true
+    };
+  }
+
+  function NotificationCardCtrl($state, Notification) {
+    var ctrl = this;
+
+    ctrl.isNewDiscussion = isNewDiscussion;
+    ctrl.isNewComment = isNewComment;
+
+    function isNewDiscussion() {
+      return ctrl.notification.kind === 'new_discussion';
```

Comment:
Not sure this knowledge should be here?
[Tell Don't Ask](https://pragprog.com/articles/tell-dont-ask) ;)